.gitlab-ci.yml

```diff
@@ -43,17 +45,23 @@ test:smoke-build:weak:
   image: debian:12-slim
   needs: []
   before_script:
-    - apt-get update && apt-get install -y git cmake libcurl4-openssl-dev
+    - apt-get update && apt-get install -yqq
+      cmake git make pkg-config python3
+      libcjson-dev
   script:
     - cmake -C cmake/CMake_weak_defaults.txt -B build tests/smoke
     - cmake --build build --parallel $(nproc --all)
     - ./build/mender-mcu-client.elf --help
 
 test:unit:
   stage: test
-  image: debian:12
+  image: debian:12-slim
   before_script:
-    - apt update && apt install -yyq g++ cmake git make lcov pkg-config curl
+    - apt update && apt install -yyq
```

Comment:
I don't think it matters much, but isn't `apt-get` preferred for scripts?

Reply:
It is. But we have this everywhere already. I believe folks in Ubuntu won't dare to change this now after many years...